src/views/AgmExtension.vue

```diff
@@ -393,8 +403,7 @@ export default class AgmExtension extends Mixins(CommonMixin, DateMixin,
       header: {
         name: FilingTypes.AGM_EXTENSION,
         certifiedBy: this.certifiedBy || '',
-        date: this.getCurrentDate, // NB: API will reassign this date according to its clock
-        waiveFees: true // *** TODO: should not do this?
```

Comment:
We probably need to add `header.header['waiveFees'] = true`.
https://github.com/bcgov/business-filings-ui/blob/c1a4f5c9039d5b78136b4849a3d6a13e78d0f080/src/views/AgmLocationChg.vue#L536

Reply:
I tested this (on AGM Location Chg) and it works fine without it.

In fact, this is a staff-only field.

In this case (and also where the fee summary data is initialized), we do not want to force "waive fees". We want to use the fee code from the Pay DB (which is $0). This should allow a future change to the fee with little or no code changes.

Reply:
You're absolutely right. Keeping this in mind for future no-fee filings. Thanks Sev.